modules/ROOT/pages/endpoints.adoc

```diff
@@ -38,11 +38,11 @@ The configuration of Partner Manager endpoints depends on the endpoint type and
 
 == Endpoint Runtime Services
 
-Using dynamic configurations, Partner Manager deploys generic endpoint services that can receive data from or send data to different endpoint configurations. Endpoint configurations of all supported protocols can be reused across various message flows and trading partners.
+Using dynamic configurations, Partner Manager deploys generic endpoint services that can receive data from or send data to different endpoint configurations. Endpoint configurations of all supported protocols can be reused across various message flows and partners.
```

Comment:
You can reuse endpoint configurations of all supported protocols across various message flows and partners.

Reply:
OK